db/migrate/20190513143015_add_theme_id_to_javascript_cache.rb

```diff
@@ -22,7 +22,8 @@ def down
   private
 
   def make_changes
-    add_reference :javascript_caches, :theme, foreign_key: { on_delete: :cascade }
+    add_column :javascript_caches, :theme_id, :bigint, null: true
+    add_index :javascript_caches, :theme_id
     add_foreign_key :javascript_caches, :theme_fields, on_delete: :cascade
```

Comment:
Is this a reason why this was not removed too? Or is it dropped in a future migration?

Reply:
It does add a foreign key constraint but I’m dropping it in a future migration.